src/Service/CaseDeadlineStatuses.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Service;
+
+abstract class CaseDeadlineStatuses
+{
+    public const HEARING_DEADLINE_EXCEEDED = 1;
+    public const PROCESS_DEADLINE_EXCEEDED = 2;
+    public const BOTH_DEADLINES_EXCEEDED = 3;
+    public const NO_DEADLINES_EXCEEDED = 4;
+}
```

Comment:
When are we upgrading to PHP 8.1? (https://stitcher.io/blog/php-enums)

Reply:
Good question, would be nice for enums.